File: archive/s/scala/even-odd.scala
// Scala Program to check if input number is Odd or Even
object evenodd 
{ 

	def check_even_odd(n: Int): Int = { 
		result = if (num%2==0) "Even" else "Odd"
    return result
	} 

	// Driver Code 
	def main(args: Array[String]) 
	{   val m= args(0).toInt
		println(evenodd(m)) 
	} 

} 

File: archive/s/swift/even-odd.swift
// for System.exit()
import Foundation


/*
    Check if there is exactly one argument and if it can be parsed as an integer
*/
guard CommandLine.argc == 2, let number = Int(CommandLine.arguments[1]) else {
    print("Usage: please input a number")
    exit(0)
}

let is_even = number % 2 == 0

if is_even {
    print("Even")
} else {
    print("Odd")
}